kas/include/tegra-jetpack4.yml

```diff
@@ -1,20 +1,23 @@
 header:
-  version: 11
+  version: 14
   includes:
   - kas/include/tegra-base.yml
 
 repos:
   meta-tegra:
-    refspec: kirkstone-l4t-r32.7.x
+    #branch: kirkstone-l4t-r32.7.x
```

Comment:
Is this supposed to be commented?

Reply:
@oldgiova thanks, that is actually intended. The concept is: "when the commit SHA refers to a branch which ist *not* the canonical release one, a comment shows the used one". So the canonical release branch is `kirkstone`, and almost all layers adhere to it, just this for example needs the `kirkstone-l4t-r32.7.x` branch to build.